src/backend/utils/misc/guc_gp.c

```diff
@@ -3215,6 +3216,16 @@ struct config_int ConfigureNamesInt_gp[] =
 		NULL, NULL, NULL
 	},
 
+	{
+		{"gp_random_insert_segments", PGC_USERSET, CUSTOM_OPTIONS,
```

Comment:
Also update in src/include/utils/sync_guc_name.h?

Reply:
Already done.